src/commands/divide.mjs

```diff
@@ -6,7 +6,7 @@
  */
 export function divide(a, b) {
   if (b === 0) {
-    throw new Error("Division by zero is not allowed");
+    throw new Error("Cannot divide by zero");
```

Comment:
it changed this to make the test pass, but why did it change the test assertion in the first place? lol